DigitalLearningSolutions.Web/Controllers/TrackingSystem/CentreConfiguration/RegistrationPromptsController.cs

```diff
@@ -1,18 +1,24 @@
 ﻿namespace DigitalLearningSolutions.Web.Controllers.TrackingSystem.CentreConfiguration
 {
+    using System;

```

Comment:
Could we add the new page to the UI tests + maybe write some controller tests on checking that TempData gets saved properly (I think ResetPW controller has these tests as an example)

Reply:
I was planning on unit testing the controller stuff as part of the final sub task when all the controller methods are redirecting to the correct places and the structure of the temp data is sorted. I'm also not sure which example tests you mean, there doesn't appear to be anything relevant to this in any of the controller tests.

Reply:
Web.Tests/Controllers/ResetPasswordControllerTests checks whether TempData gets set as expected :)